samples/dynatablerf/pom.xml

```diff
@@ -94,6 +94,18 @@
       </exclusions>
     </dependency>
 
+    <!-- added to work with GWT 2.11.0 and newer --> 
```

Comment:
Doesn't this provide classes using jakarta package and some imports need to be updated as well somewhere?

Reply:
After adding the dependencies I were able to run the sample with Java 11 & 17. In case we switch to **jakarta.xml.bind**, we might have code changes.

Reply:
Or to be more precise: jaxb-impl:4.x is Jakarta EE 10 (based on jakarta package) and I am wondering if it really matches jaxb-api:2.4.x which should be Jakarta EE 8 (using the javax package).